### ISSUE TITLE
Local build is failing due to missing  gcr.io image

### ISSUE THREAD

**Issue by entropyie** [kubernetes-sigs/kubeadm-dind-cluster#129]
Hi, I am building locally due to an issue with Centos 7: https://github.com/kubernetes/kubernetes/issues/43704 that breaks the pre-built images.

However I can't build locally because some needed container images appear to be unavailable:

```
[root@centos7-dev kubeadm-dind-cluster]# ./build/build-local.sh 
Sending build context to Docker daemon 44.54 kB
Step 1 : FROM gcr.io/kubeadm/ci-xenial-systemd:base-master-6f6ae74260bfa1b6d97251e6e0191489aeff0939
Trying to pull repository gcr.io/kubeadm/ci-xenial-systemd ... 
Pulling repository gcr.io/kubeadm/ci-xenial-systemd
unauthorized: authentication required
```

Can you restore 
`gcr.io/kubeadm/ci-xenial-systemd:base-master-6f6ae74260bfa1b6d97251e6e0191489aeff0939`
or update the script to pull a currently available container?

Thanks!

**Comment by ivan4th** [comment]
Hi, the fix is currently in `fix-base-image` branch: https://github.com/kubernetes-sigs/kubeadm-dind-cluster/tree/fix-base-image
I'll merge it to master once I have CI issues sorted out after the repo was moved (CircleCI is not enabled for `kubernetes-sigs` org)

**Comment by ivan4th** [comment]
Fixed by #130
